engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/ConfluentSchemaRegistryProvider.scala

```diff
@@ -37,15 +37,15 @@ object ConfluentSchemaRegistryProvider extends Serializable {
   def apply(schemaRegistryClientFactory: ConfluentSchemaRegistryClientFactory): ConfluentSchemaRegistryProvider =
     avroPayload(
       schemaRegistryClientFactory,
-      formatKey = false
+      formatKey = true
```

Comment:
Why not to just remove formatKey flag? Or you want to make it in next PR with better test generator?

Reply:
That exact change I do in separate PR. Here goes the minimal change that introduces #inputMeta & ConsumerRecord to avro sources.

Reply:
Removed `formatKey` params.